### PR TITLE
Added instructions and UI test for test Bugzilla44096

### LINKED ISSUE #2378: UI test B44096 needs to be automated
- [ ] Test needs to be automated
- [ ] Manual test needs instructions

### PR DESCRIPTION
**Pull request by mikescandy**
### Description of Change ###

Added instructions and UI test for test Bugzilla44096
I updated the test, including a nested button inside the views. From the bug description, it seemed to me that the problem was that nested controls were still clickable after disabling the parent view.
In order to test click on both nested and parent view, i had to be a bit creative with layouts and click.
Also one thing: is it correct that transparent views will not trigger clicks on UWP? This behavior is different on Android.

### Issues Resolved ### 

- fixes #2378

### API Changes ###

 None

### Platforms Affected ### 
- Core/XAML (all platforms)
- Android
- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### PR Checklist ###

- [X] Has automated tests
- [X] Rebased on top of the target branch at time of PR
- [X] Changes adhere to coding standard

### CODE REVIEW

**Comment by PureWeen** [comment]
The four added tests are all failing on Android

> Also one thing: is it correct that transparent views will not trigger clicks on UWP? This behavior is different on Android.

Yea. How all the platforms handle transparent views and click propagation are all different and UWP isn't quite in sync. I wrote up an analysis of all of that here when trying to figure out how button clicking should work. That issue is more around it being enabled but there's definitely some symmetry between platforms that needs to happen

https://github.com/xamarin/Xamarin.Forms/issues/1454#issuecomment-406701638

**Comment by mikescandy** [comment]
😠 i'll have a look. sorry!

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla44096.cs [hunk 1]
```diff
@@ -9,48 +9,74 @@
 
 namespace Xamarin.Forms.Controls.Issues
 {
-#if UITEST
-	[Category(UITestCategories.IsEnabled)]
-#endif
-
 	[Preserve(AllMembers = true)]
```
Comment (hartez) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla44096.cs]: Why remove the test category?

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla44096.cs [hunk 2]
```diff
@@ -139,65 +187,74 @@ void AddTapGesture(Label result, View view)
...
 				{
-					result.Text = Child;
+					result.Text = view.AutomationId;
 				})
 			};
 			view.GestureRecognizers.Add(tapGestureRecognizer);
 		}
 
-#if UITEST
+#if UITEST && (__WINDOWS__ || __ANDROID__)
```
Comment (hartez) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla44096.cs]: Why is this no longer run on iOS?

**Comment by samhouts** [comment]
@mikescandy Thank you for your contribution! We would like to accept this, but it does require your attention. Please review the comments made above and see if you can make the requested changes. If so, please reopen this PR with your new commits. Thank you!

**Pull request closed without merging**